app/models/job_application.rb

```diff
@@ -16,6 +16,7 @@ class JobApplication < ApplicationRecord
 
   has_one_attached :cover_letter
   has_one_attached :resume
+  has_one :application_criterion, through: :job
```

Comment:
Will this work with the user having many jobs?

Reply:
Yes. Also job_application && application_criteria have job_id

Reply:
![image](https://github.com/user-attachments/assets/d3ec2a5c-a6f5-4601-966f-0b2fbd8cf990)

Can you check?

Reply:
![image](https://github.com/user-attachments/assets/7579bcd1-53a5-4c43-8bf0-f8c77b54e6eb)

a user has many job_applications ... but a job application belongs to one job (only) and a job has 1 application_criterion only ... I see it as an inner join ...

just a shortcut instead of doing job_application.job.application_criterion ...